records/README.md

```diff
@@ -0,0 +1,220 @@
+# IPRS - InterPlanetary Record System spec
+Authors: [Juan Benet](github.com/jbenet)
+
+Reviewers:
+
+- [Kristoffer Strom](github.com/krl)
+- [Jeromy Johnson](github.com/whyrusleeping)
+
+* * *
+
+The [Spec](../) for IPRS.
+
+This spec defines IPRS (InterPlanetary Record System) a system for distributed record keeping meant to operate across networks that may span massive distances (>1AU) or suffer long partitions (>1 earth yr).
+IPRS is meant to establish a common record-keeping layer to solve common problems. It is a well-layered protocol: it is agnostic to underlying replication and transport systems, and it supports a variety of different applications.
+IPRS is part of the InterPlanetary File System Project, and is general enough to be used in a variery of other systems.
+
+## Definitions
+
+### Records
+
+A (distributed) `record` is a piece of data meant to be transmitted and stored in various computers across a network. It carries a `value` external to the record-keeping system, which clients of the record system use. For example: a namespace record may store the value of a name. All record systems include a notion of _record validity_, which allows users of the record system to verify whether the value of a record is correct and valid under the user's circumstances. For example, a record's validity may depend upon a cryptographic signature, a range of spacetime, et-cetera.
+
+### Record System
+
+A (distributed) `record system` is a protocol which defines a method for crafting, formating, distributing, verifying, and using records over computer networks. (e.g. the Domain Name System).
+
+
+- `crafting` - construction of a record (the process of calculating the values of a record)
+- `formating` - serializing of a record into a bitstring.
+- `distributing` - transportation of a record from one set of computers to another.
+- `verifying` - checking a record's values to ensure correctness and validity.
+
+
+### Validity Schemes
+
+A `validity scheme` is a small sub-protocol that defines how a record's _validity_ is to be calculated. `Validity` is the quality of a record being usable by a user at a particular set of circumstances (e.g. a range of time). It is distinct from `correctness` in that `correctness` governs whether the record was correctly constructed, and `validity` governs whether a record may still be used. All _valid_ records must be _correct_. For simplicity, the process of checking correctness is included in the `validity scheme`.
+
+For example, suppose Alice and Bob want to store records on a public bulletin board. To make sure their records are not tampered with, Alice and Bob decide they will include cryptographic signatures. This can ensure correctness. Further, they also agree to add new records every day, to detect whether their records are being replayed or censored. Thus, their `validity scheme` might be:
```

Comment:
yeah, you're right. in general i find "validity scheme" weak and confusing. i'll try to bring these bullets in.